sympy/functions/elementary/tests/test_hyperbolic.py

```diff
@@ -521,6 +533,10 @@ def test_asech():
     assert asech(2) == I*pi/3
     assert asech(-2) == 2*I*pi / 3
 
+    # at infinites
+    assert asech(oo) == I*pi/2
+    assert asech(-oo) == I*pi/2
```

Comment:
If you intend on doing this, move all from `def test_asech_infinities():`and remove that.